spec/bolt/outputter/human_spec.rb

```diff
@@ -40,6 +40,32 @@
     TABLE
   end
 
+  it "formats a task" do
+    task = {
+      'name' => 'cinnamon roll',
+      'description' => 'A delicious sweet bun',
+      'usage' => 'Nourishment and happiness',
+      'parameters' => {
+        'icing' => {
+          'type' => 'Cream cheese',
+          'description' => 'Rich, tangy, sweet'
+        }
+      }
+    }
+    outputter.print_task(task)
+    expect(output.string).to eq(<<-TASK_OUTPUT)
+cinnamon roll - A delicious sweet bun
+
+USAGE:
+Nourishment and happiness
+
+PARAMETERS:
+- icing: Cream cheese
+\tRich, tangy, sweet
```

Comment:
Also gross. `\t` just seemed like the best way to make sure editors don't accidentally break this test (as mine is prone to do with tabs). But definitely tell me if it's unacceptable.

Reply:
why not just use spaces?